src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/UnprotectedAttributeTests.cs

```diff
@@ -306,7 +306,7 @@ public static void TestUnprotectedAttributes1_EmptySet()
             EnvelopedCms ecms = new EnvelopedCms();
             ecms.Decode(encodedMessage);
             
-            Assert.Equal(1, ecms.UnprotectedAttributes.Count);
+            Assert.Equal(2, ecms.UnprotectedAttributes.Count);
```

Comment:
We can only really know they'll come out in certain order because of how we coded them, there's no real guarantee of the order, which is why I didn't include any further testing. The ideas that I came with on the spot wouldn't give too much information on the assertion messages but I'll post one that better tests the wanted condition.